### PR TITLE
Infoblox

### PR DESCRIPTION
**Pull request by ShahafBenYakir**
## Status
Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/17814

## Description
Added Infoblox new integration

## Required version of Demisto
x.x.x

## Does it break backward compatibility?
   - No

## Must have
- [x] Tests
- [ ] Documentation (with link to it)
- [ ] Code Review

## Technical writer review
Mention and link to the files that require a technical writer review.
- [ ] [YAML file](https://github.com/demisto/content/blob/34a44cf7c5110bd16c79a88b9eca0abeebf24b37/Integrations/Infoblox/Infoblox.yml)

### CODE REVIEW

File: Tests/test_utils.py [hunk 1]
```diff
@@ -113,6 +113,7 @@ def filter_packagify_changes(modified_files, added_files, removed_files, tag='ma
     for file_path in added_files:
         if file_path.split("/")[0] in PACKAGE_SUPPORTING_DIRECTORIES:
             with open(file_path) as f:
+                print(file_path)
```
Comment (yuvalbenshalom) [inline Tests/test_utils.py]: I don't think this should be here

File: Integrations/Infoblox/Infoblox.py [hunk 2]
```diff
@@ -0,0 +1,960 @@
...
+    INTEGRATION_NAME:
+        Name of the integration as shown in the integration UI, for example: Microsoft Graph User.
+
+    INTEGRATION_COMMAND_NAME:
+        Command names should be written in all lower-case letters,
+        and each word separated with a hyphen, for example: msgraph-user.
+
+    INTEGRATION_CONTEXT_NAME:
+        Context output names should be written in camel case, for example: MSGraphUser.
+"""
```
[the remark this answers is not in the document]
Comment (ShahafBenYakir) [reply Integrations/Infoblox/Infoblox.py]: True

File: Tests/test_utils.py (hunk 1 above)
Comment (ShahafBenYakir) [reply Tests/test_utils.py]: So ashamed 😢

**Comment by richardbluestone** [comment]
yml file reviewed and updated.

**Pull request merged**